tests/components/tibber/test_services.py

```diff
@@ -0,0 +1,254 @@
+                }
+            },
+        ),
+        MagicMock(
+            name="second_home",
+            info={
+                "viewer": {
+                    "home": {
+                        "currentSubscription": {
+                            "priceInfo": {
+                                "today": [
+                                    {
+                                        "startsAt": today.isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                    {
+                                        "startsAt": (
+                                            today + dt.timedelta(hours=1)
+                                        ).isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                ],
+                                "tomorrow": [
+                                    {
+                                        "startsAt": tomorrow.isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                    {
+                                        "startsAt": (
+                                            tomorrow + dt.timedelta(hours=1)
+                                        ).isoformat(),
+                                        "total": 0.46914,
+                                        "level": "VERY_EXPENSIVE",
+                                    },
+                                ],
+                            }
+                        }
+                    }
+                }
+            },
+        ),
+    ]
+    mock_homes[0].name = "first_home"
+    mock_homes[1].name = "second_home"
+    return mock_homes
+
+
+def create_mock_tibber_connection():
+    """Create a mock tibber connection."""
+    tibber_connection = MagicMock()
+    tibber_connection.get_homes.return_value = generate_mock_home_data()
+    return tibber_connection
+
+
+def create_mock_hass():
+    """Create a mock hass object."""
+    mock_hass = MagicMock
+    mock_hass.data = {"tibber": create_mock_tibber_connection()}
+    return mock_hass
+
+
+def remove_microseconds(dt):
+    """Remove microseconds from a datetime object."""
+    return dt.replace(microsecond=0)
+
+
+async def test_get_prices():
+    """Test __get_prices with mock data."""
+    today = remove_microseconds(dt.datetime.now())
+    tomorrow = remove_microseconds(dt.datetime.now() + dt.timedelta(days=1))
+    call = ServiceCall(
+        DOMAIN,
+        PRICE_SERVICE_NAME,
+        {"start": today.date().isoformat(), "end": tomorrow.date().isoformat()},
+    )
+
+    result = await __get_prices(call, hass=create_mock_hass())
```

Comment:
Please don't test the service handler directly. Call the service via the service registry in core after setting up the integration via a mock config entry.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations